### ISSUE TITLE
Rectify GAN Training Problems

### ISSUE THREAD

**Issue by Akettle44** [Akettle44/SRGAN-PyTorch#23]
Use ganhacks page to alter the training loop to generate better results / FID score. 

Subtasks:
- Change perceptual loss network back to VGG19
- Add mixed precision training (to speed up computations)
- Modify training loop
